android/app/build.gradle

```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:0.19.+"
+    compile "com.facebook.react:react-native:+"
```

Comment:
I think "0.19.+" means "minor version 19, any patch version". Before I made this change, I was getting compiler errors.